SecurityPkg/Include/Library/Tpm2HelpLib.h

```diff
@@ -0,0 +1,185 @@
+/** @file
+  Definitions for TPM2 helper functions
+
+Copyright (c) 2013 - 2024, Intel Corporation. All rights reserved. <BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _TPM2_HELP_LIB_H_

```

Comment:
nit, current style guidelines have

```suggestion
#ifndef TPM2_HELP_LIB_H_

```